geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/api/MembershipLocator.java

```diff
@@ -38,7 +38,7 @@
 
   void restarting() throws IOException;
 
-  SocketAddress getBindAddress();
+  SocketAddress getSocketAddress();
```

Comment:
since the server socket has to be bound before it has an address, it makes sense not to refer to this as the "bind address" ✓